addons/website/models/ir_qweb.py

```diff
@@ -124,7 +124,8 @@ def _post_processing_att(self, tagName, atts):
         website = ir_http.get_request_website()
         if not website and self.env.context.get('website_id'):
             website = self.env['website'].browse(self.env.context['website_id'])
-        if website and tagName == 'img' and 'loading' not in atts:
+        if website and tagName == 'img' and 'loading' not in atts \
+                and 'src' in atts and '/logo/' not in atts['src']:
```

Comment:
Nitpicking: Could have been a more specific check, also adding `'/logo/' not in atts['src']` alongside the logo check, to not match other logo fields ?

Just adding this nitpicking one as I commented the first commit anyway

Reply:
> Could have been a more specific check

I don't see how I could.

> Just adding this nitpicking one as I commented the first commit anyway

I am not a fan of this commit either and the benefit is 0 to me, I kept it as we will remove this code in the future IMO (as explained in my commit message) but we can remove it.

Reply:
> I don't see how I could (EDIT: well I guess a regex but then this commit is really not worth it :'))

I was just speaking about adding
```py
and 'src' in atts and '/logo/' not in atts['src'] and '/website/' not in atts['src']:
```
Instead of just
```py
and 'src' in atts and '/logo/' not in atts['src']:
```
To not match any image that would be loading the logo field of another model (nitpicking I know, just thought it was low effort to add it).

Maybe I am missing something tho ;-)

Reply:
When I originally made the commit I thought that other logo images could benefit from this to: they are likely not big images and lazy loading is just a cause of flickering for no reason in that case IMO (and in the standard codebase, there are only 3 logo fields, 2 of which not used in the frontend).

But I was not a fan of adding a processing `XXX not in machin` for all images anyway, so let's remove this commit. We'll make something better when we'll review the automatic lazy loading.

Reply:
Commit removed :+1: